### PR TITLE
[storage][front-end] Remove xxxDeleteResponse messages

### PR DESCRIPTION
**Pull request by glimchb**
Following Google [AIP-135](https://google.aip.dev/135) - Standard methods: Delete

Related to #174

### CODE REVIEW

**Review by llabordehpe — APPROVED** [review]
Changes look good to me.

**Review by jainvipin — APPROVED** [review]
thanks @glimchb - this change looks good to me and aligned with standardized APIs.

**Review by mestery — APPROVED** [review]

File: storage/v1/frontend_nvme_pcie.proto [hunk 1]
```diff
@@ -8,14 +8,15 @@ option go_package = "github.com/opiproject/opi-api/storage/v1/gen/go";
...
+import "google/protobuf/empty.proto";
 
 // Front End (host-facing) APIs.
 
 // NVMe/PCIe emulation
 
 service NVMeSubsystemService {
     rpc NVMeSubsystemCreate (NVMeSubsystemCreateRequest) returns (NVMeSubsystem) {}
-    rpc NVMeSubsystemDelete (NVMeSubsystemDeleteRequest) returns (NVMeSubsystemDeleteResponse) {}
+    rpc NVMeSubsystemDelete (NVMeSubsystemDeleteRequest) returns (google.protobuf.Empty) {}
```
Comment (PWAlessi) [inline storage/v1/frontend_nvme_pcie.proto:19]: 👍 better to return `empty` than an empty message

**Review by PWAlessi — APPROVED** [review]
👍 this is great progress.

**Review by benlwalker — APPROVED** [review]

**Review by mkalderon — APPROVED** [review]
looks good. thanks

**Pull request merged**